### DISCUSSION TITLE
S3 public bucket access is set to "Block public access" by default

### DISCUSSION THREAD

**Discussion by fleeser** [serverpod/serverpod#1000] [General]
I was trying to get File uploads working when I encountered a problem:
Whenever I tried to get an upload description and upload files it failed and I did not know why.
When I looked into the bucket settings of the automatically created public S3 bucket by the terraform scripts, I found that the access is always blocked. After disabling **Block all public access** I was able to upload files successfully.

The documentation does not state anything about the access settings or anything like that.

So my question here is if this is done on purpose or if there is a lack of documentation. 

Also there are some underlying options that you can choose like **Block public access to buckets and objects granted through new access control lists (ACLs)**, **Block public access to buckets and objects granted through any access control lists (ACLs)** etc. So I would also like to know if any of these options should be marked? For now, I disabled every single of these options.

**Comment by vlidholt** [comment]
This may be an issue with the terraform scripts. I'm adding this as an issue.
